readme.md

```diff
@@ -347,7 +347,7 @@ Lifting is when you take a value and put it into an object like a [functor](#poi
 Some implementations have a function called `lift`, or `liftA2` to make it easier to run functions on functors.
 
 ```js
-const liftA2 = (f) => (a, b) => a.map(f).ap(b);
+const liftA2 = (f) => (a, b) => a.map(f).map(b);
```

Comment:
This is like the 3rd time people thought it was a typo, we need to add a note there?